.travis.yml

```diff
@@ -1,5 +1,9 @@
 language: java
 
+jdk:
+  - oraclejdk8
+  - oraclejdk9
```

Comment:
This will not pass until Presto is fully compatible, which means builds will fail for a long time. Also, does this apply to the runtime or to build time, too? Supporting build time will take even longer.

Reply:
This was just for me testing. I'll move it to a different PR.

Reply:
This switches for the whole job, but for just tests, we should be able to manually configure the matrix and use `jdk_switcher` after the build part: https://docs.travis-ci.com/user/languages/java/#Switching-JDKs-Within-One-Job